WrapperClass.java

```diff
@@ -1 +1,21 @@
+package com.main;
 
+/**
+ Simple program to compare variables of wrapper class
+ Practical 9 from core java practicals
+ @author Mashuk Patel
+ */
+public class WrapperClass {
+
+	public static void main(String[] args) {
+		Integer a=new Integer(30);
```

Comment:
1) The variable name must be meaningful.
2) Space must be applied after the variable.
Integer a = new Integer(30);